codex_of_the_damned/templates/archetypes/index.html

```diff
@@ -681,4 +681,27 @@ <h2>Bibliodèque</h2>
 		</tbody>
 	</table>
 </div>
+
+{% trans trimmed
+github=external("https://github.com/lionel-panhaleux/codex-of-the-damned", "Github")
+%}
+<h2>Players Contributions</h2>
+<p>
+	As the game develops, new archetypes are built by players. This section is a way for
+	everyone to share decklists they are comfortable with. Anyone that wants to share a
+	decklist is welcomed to do so using {{ github }}. The Codex team will then review it
```

Comment:
```suggestion
	decklist is welcome to do so using {{ github }}. The Codex team will then review it
```
welcomed (verb, passive) and welcome (adjective) don't have the same meaning, see https://thegrammarexchange.infopop.cc/topic/you-are-not-welcome-here-anymore